C/cp/Activity_selection.c

```diff
@@ -0,0 +1,85 @@
+// C Code for Activity Selection
+//Greedy Algorithm approach
+#include <stdio.h>
+#include <conio.h>
+//swap function to check and swap values
+void swap(int *x,int *y)
+{
+	int temp;
+	temp=*x;
+	*x=*y;
+	*y=temp;
+}
+//Main function start
+int main()
+{
+	int num,i,j,prev=0,count=1;
+	int *start,*finish;
+	//Dynamic Memory Allocation for arrays
+	start = (int*) malloc(num * sizeof(int));
+	finish = (int*) malloc(num * sizeof(int));
+	//Take number of activites as a input
+	printf("Enter the number of activities :");
+	scanf("%d",&num);
+	//Take start and finish time as a input
+	printf("\nEnter the start and finish time of activities :\n");
+	for(i=0;i<num;i++)
+	{
+		scanf("%d",&start[i]);
+		scanf("%d",&finish[i]);
+	}
+	//sorting starts here
+	//Bubble sort - compare adjacent element and swap it
+	for(i=0;i<num;i++)
+	{
+		//In bubble sort, maximum value is placed at last index
+		//In each pass we left the last index i.e num-1-i
+		for(j=0;j<num-1-i;j++)
+		{
+			if(finish[j]>finish[j+1])
+			{
+				//call swap function
+				//pass by address
+				swap(&finish[j],&finish[j+1]);
+				swap(&start[j],&start[j+1]);
+			}
+		}
+	}
+	//condition for Activity selection
```

Comment:
please use multi line comments here.

Reply:
by mistake i deleted the branch before merging thats y i am not be able to make changes. Can you tell me how to do or i have to make a new PR? @Kajol-Kumari